New `maintenance.timezone` config requires IANA Time Zone database

### Describe the bug

The [official Gatus container image](https://hub.docker.com/r/twinproduction/gatus) is missing the required timezone data to be able to properly interpret timezone values.

When e.g. setting `maintenance.timezone = "Europe/Zurich"`, Gatus panics with:

```
panic: invalid timezone specified or format not supported. Use IANA timezone format (e.g. America/Sao_Paulo): unknown time zone Europe/Zurich
```

According to the [docs of `time.LoadLocation`](https://pkg.go.dev/time#LoadLocation):

> LoadLocation looks for the IANA Time Zone database in the following locations in order:
> 
> - the directory or uncompressed zip file named by the ZONEINFO environment variable
> - on a Unix system, the system standard installation location
> - $GOROOT/lib/time/zoneinfo.zip
> - the time/tzdata package, if it was imported

A simple `apk install tzdata` fixes the panic. But importing the `time/tzdata` would also solve the issue.

### What do you see?

```
panic: invalid timezone specified or format not supported. Use IANA timezone format (e.g. America/Sao_Paulo): unknown time zone Europe/Zurich
```

### What do you expect to see?

No panic.

### List the steps that must be taken to reproduce this issue

Set `maintenance.timezone = "Europe/Zurich"`.

### Version

5.12.0

### Additional information

_No response_

--- a/config/maintenance/maintenance.go
+++ b/config/maintenance/maintenance.go
@@ -6,6 +6,7 @@ import (
 	"strconv"
 	"strings"
 	"time"
+	_ "time/tzdata" // Required for IANA timezone support
 )
 
 var (
@@ -52,7 +53,7 @@ func GetDefaultConfig() *Config {
 }
 
 // IsEnabled returns whether maintenance is enabled or not
-func (c Config) IsEnabled() bool {
+func (c *Config) IsEnabled() bool {
 	if c.Enabled == nil {
 		return true
 	}
@@ -101,7 +102,7 @@ func (c *Config) ValidateAndSetDefaults() error {
 }
 
 // IsUnderMaintenance checks whether the endpoints that Gatus monitors are within the configured maintenance window
-func (c Config) IsUnderMaintenance() bool {
+func (c *Config) IsUnderMaintenance() bool {
 	if !c.IsEnabled() {
 		return false
 	}
@@ -127,7 +128,7 @@ func (c Config) IsUnderMaintenance() bool {
 	return now.After(startOfMaintenancePeriod) && now.Before(endOfMaintenancePeriod)
 }
 
-func (c Config) hasDay(day string) bool {
+func (c *Config) hasDay(day string) bool {
 	for _, d := range c.Every {
 		if d == day {
 			return true

